
<?xml version='1.0'?>

    <robot name="robot">

    <material name="orange">
      <color rgba="1.0 0.5 0.2 1" />
    </material>
    <material name="gray">
      <color rgba="0.2 0.2 0.2 1" />
    </material>
    <material name="blue">
      <color rgba="0 0.5 1 1" />
    </material>

    <link name="base_link" />

    <link name="imu_link">
      <visual>
        <geometry>
          <box size="0.06 0.04 0.02" />
        </geometry>
        <material name="orange" />
      </visual>
    </link>

    <link name="laser_link">
      <visual>
        <geometry>
          <box size="0.05 0.05 0.07" />
        </geometry>
        <material name="gray" />
      </visual>
    </link>


    <joint name="imu_link_joint" type="fixed">
      <parent link="base_link" />
      <child link="imu_link" />
      <origin xyz="0 0 0" />
    </joint>

    <joint name="laser_link_joint" type="fixed">
      <parent link="base_link" />
      <child link="laser_link" />
      <origin xyz="0.23 0.0 0.452" />
    </joint>

</robot>

